argilla-sdk/src/argilla_sdk/responses.py

```diff
@@ -49,10 +59,14 @@ def __init__(
         if user_id is None:
             raise ValueError("user_id is required")
 
+        if isinstance(status, str):
```

Comment:
Should we warn the user when they create responses with a status of submitted and a different user_id?

Reply:
@burtenshaw I would say no. I think people should consciously change these kinds of things and probably know what they are doing in that case.